syntax = "proto3";

package trip;

service TripService {
  rpc CreateTrip (CreateTripRequest) returns (TripResponse);
  rpc ListTrips (Empty) returns (TripListResponse);
}

message Empty {}

message CreateTripRequest {
  string destination = 1;
}

message TripResponse {
  string id = 1;
  string destination = 2;
}

message Trip {
  string id = 1;
  string destination = 2;
}

message TripListResponse {
  repeated Trip trips = 1;
}
